src/internal/components/cartesian-chart/__tests__/left-labels.test.tsx

```diff
@@ -0,0 +1,76 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+import { render } from '@testing-library/react';
+import LeftLabels from '../../../../../lib/components/internal/components/cartesian-chart/left-labels';
+import {
+  getVisibleTicks,
+  getLabelBBox,
+} from '../../../../../lib/components/internal/components/cartesian-chart/label-utils';
+import { NumericChartScale } from '../../../../../lib/components/internal/components/cartesian-chart/scales';
+
+jest.mock('../../../../../lib/components/internal/components/cartesian-chart/label-utils', () => ({
+  ...jest.requireActual('../../../../../lib/components/internal/components/cartesian-chart/label-utils'),
+  getVisibleTicks: jest.fn().mockReturnValue([]),
+  getLabelBBox: jest.fn().mockReturnValue({ width: 0, height: 0 }),
+}));
+
+jest.mock('../../../../../lib/components/internal/components/responsive-text', () => ({
+  __esModule: true,
+  default: () => <text>...</text>,
+}));
+
+describe('cartesian chart left labels', () => {
+  beforeEach(() => {
+    (getVisibleTicks as any).mockClear();
+  });
+
+  test('all ticks are visible', () => {
+    (getVisibleTicks as any as jest.Mock).mockReturnValueOnce([
```

Comment:
```suggestion
    jest.mocked(getVisibleTicks).mockReturnValueOnce([
```

This helper changes nothing in runtime but makes the code more readable